snippets/client-platforms/cyclejs/setup.md

```diff
@@ -0,0 +1,14 @@
+```js
+import {createHistory} from "history";
+import {makeRouterDriver} from 'cyclic-router'
+import {makeAuth0Driver, protect} from "cyclejs-auth0";
+
+function main(sources) {
+    //your application's code
+}
+
+const drivers = {
+    auth0: makeAuth0Driver("client-ID", "domain"),
```

Comment:
We should use the code that automatically generates the user's ID and domain here :) 

`auth0: makeAuth0Driver('<%= account.clientId %>', '<%= account.namespace %>')`